components/auth/org.wso2.carbon.auth.oauth/src/main/java/org/wso2/carbon/auth/oauth/configuration/OAuthConfigurationService.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.wso2.carbon.auth.oauth.configuration;
+
+import org.wso2.carbon.auth.oauth.configuration.models.OAuthConfiguration;
+
+/**
+ * Configuration holder for oAuth component
```

Comment:
Camel case error in word oAuth

Reply:
@prasa7 Do we have to follow camel case in java doc as well?

Reply:
Yes, it should be OAuth.